modules/ROOT/pages/bolt-compatibility.adoc

```diff
@@ -2,6 +2,22 @@
 
 = Bolt Protocol and Neo4j compatibility
 
+:fn-calver: footnote:calver[This includes (newer) calver releases newer such as `2026.01`. \
+At some point, the server might drop support for older Bolt versions, but there's no plan or guarantee for when or if that will happen. \
+As soon as a decision is made, it will be documented here.]
+
+== Bolt 6.x
+
+[cols="^,^",options="header"]
+|===
+| Neo4j version
+| Bolt `6.0`
+
+| `2025.07`+{fn-calver}
```

Comment:
Should the footnote really be here? If, say, 2025.11 would _not_ support 6.0, I assume we'd have a new line in the table for `2025.11+` that lacks compatibility with 6.0. In that case, we don't need the footnote.